server/start.go

```diff
@@ -452,62 +448,12 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 	}
 
 	// At this point it is safe to block the process if we're in gRPC only mode as
-	// we do not need to start Rosetta or handle any Tendermint related processes.
+	// we do not need to handle any Tendermint related processes.
 	if gRPCOnly {
 		// wait for signal capture and gracefully return
 		return WaitForQuitSignals()
 	}
 
-	var rosettaSrv crgserver.Server
-	if config.Rosetta.Enable {
-		offlineMode := config.Rosetta.Offline
-
-		// If GRPC is not enabled rosetta cannot work in online mode, so we throw an error.
-		if !config.GRPC.Enable && !offlineMode {
-			return errors.New("'grpc' must be enable in online mode for Rosetta to work")
-		}
-
-		minGasPrices, err := sdktypes.ParseDecCoins(config.MinGasPrices)
-		if err != nil {
-			ctx.Logger.Error("failed to parse minimum-gas-prices: ", err)
-			return err
-		}
-
-		conf := &rosetta.Config{
-			Blockchain:          config.Rosetta.Blockchain,
-			Network:             config.Rosetta.Network,
-			TendermintRPC:       ctx.Config.RPC.ListenAddress,
-			GRPCEndpoint:        config.GRPC.Address,
-			Addr:                config.Rosetta.Address,
-			Retries:             config.Rosetta.Retries,
-			Offline:             offlineMode,
-			GasToSuggest:        config.Rosetta.GasToSuggest,
-			EnableFeeSuggestion: config.Rosetta.EnableFeeSuggestion,
-			GasPrices:           minGasPrices.Sort(),
-			Codec:               clientCtx.Codec.(*codec.ProtoCodec),
-			InterfaceRegistry:   clientCtx.InterfaceRegistry,
-		}
-
-		rosettaSrv, err = rosetta.ServerFromConfig(conf)
-		if err != nil {
-			return err
-		}
```

Comment:
where is this set now?

Reply:
Right now the only way to start rosetta server would be through simd, and the configuration would be pass through the cli arguments.

https://github.com/cosmos/cosmos-sdk/blob/main/tools/rosetta/config.go#L59

Reply:
Can you expand more on how that would look like?

Reply:
hmm, we should allow rosetta to read the app/config.toml in order to get some of these values to void needing to set many of them, but we can do that in another pr.

Reply:
> Can you expand more on how that would look like?

https://docs.cosmos.network/main/run-node/rosetta#use-rosetta-command

Reply:
> hmm, we should allow rosetta to read the app/config.toml in order to get some of these values to void needing to set many of them, but we can do that in another pr.

do you think that's necessary? I removed all that info from the [toml](https://github.com/cosmos/cosmos-sdk/pull/14062/files#diff-415fc3ca04fcbd0ed15299c290b057eec05692573e9d7f3a114a6ed1dcd7a480L152) as there are some [default values](https://github.com/cosmos/cosmos-sdk/blob/main/tools/rosetta/config.go#L20) for the cli flags

Reply:
mainly mean for everything that wasn't part of the rosetta app.toml section